Services/Dashboard/Access/DashboardAccess.php

```diff
@@ -3,28 +3,36 @@
 /**
  * This file is part of ILIAS, a powerful learning management system
  * published by ILIAS open source e-Learning e.V.
+ *
  * ILIAS is licensed with the GPL-3.0,
  * see https://www.gnu.org/licenses/gpl-3.0.en.html
  * You should have received a copy of said license along with the
  * source code, too.
+ *
  * If this is not the case or you just want to try ILIAS, you'll find
  * us at:
  * https://www.ilias.de
  * https://github.com/ILIAS-eLearning
- */
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
 
 namespace ILIAS\Dashboard\Access;
 
+use ilDBInterface;
+use ILIAS\DI\Container;
+use ilObjDashboardSettings;
+use ilRbacSystem;
+
 /**
- * Dashboard permission wrapper
- *
  * @author Alexander Killing <killing@leifos.de>
  */
 class DashboardAccess
 {
-    protected \ilRbacSystem $rbac_system;
-    protected \ilDBInterface $db;
-    protected static int $setting_ref_id = 0;
+    protected readonly ?ilRbacSystem $rbac_system;
```

Comment:
Why are the properties nullable? They where made readonly and assigned in the constructor, so they will never be null.